bokeh/models/mappers.py

```diff
@@ -50,17 +50,47 @@ class LinearColorMapper(ColorMapper):
     this are clamped to ``high``.
     """)
 
-    # TODO: (jc) what is the color code for transparent?
-    # TODO: (bev) better docstring
-    reserve_color = Color("#ffffff", help="""
-    Used by Abstract Rendering.
```

Comment:
I think we can safely remove anything to do with abstract rendering, but maybe that should be a separate PR?